goodmap/formatter.py

```diff
@@ -9,10 +10,22 @@ def safe_gettext(lista):
 
 
 def prepare_pin(place, visible_fields):
+    type_of_place = get_type_of_place(place)
     pin_data = {
         "title": place["name"],
-        "subtitle": lazy_gettext(place["type_of_place"]),
+        "subtitle": lazy_gettext(type_of_place),
         "position": place["position"],
-        "data": {gettext(field): safe_gettext(place[field]) for field in visible_fields}
+        "data": {
+            gettext(field): safe_gettext(place[field]) for field in visible_fields
+        },
     }
     return pin_data
+
+
+def get_type_of_place(place):
+    # TODO: should we require 'type_of_place'? For now, this workaround
```

Comment:
TODO should be rather task than question, but yes. We should discuss format of data. It could also be 'category' or sth?